### ISSUE TITLE
Network Status: BLOCKED  to WIFI 

### ISSUE THREAD

**Issue by canary5** [schwabe/ics-openvpn#1135]
To make issues more manageable, I would appreciate it if you fill out the following details as applicable:

# General information
1. Android 10
2. Custom ROM
3. Pixel 3 
4. 0.7.13
 
# Description of the issue
Sometimes when changing through networks its stops reconnecting and idling.
10:16 Network Status: BLOCKED  to WIFI 

But wifi always working, just sometimes it giving that announce, maybe cause of mav randomization. But its always online even blocked.

Is it possible to skip that and reconnect

**Comment by schwabe** [comment]
BLOCKED is the general network status that Android reports.

**Comment by canary5** [comment]
Yes but the report is like 1 sec when connecting to wifi. Wifi is working, but openvpn still not reconnecting and freezed under Blocked status

**Comment by schwabe** [comment]
I need some more information to reproduce this or at least a complete log

**Comment by canary5** [comment]
I have managed to reproduce. It was freezed today unknown state: no network, even wifi was online. I just did wifi 0n/off but its still didnt reconnected for same reason: wifi blocked. Wifi was 100% online. 
But if i click reconnect manually same sec all connected.

2020-01-27 10:06:25 official build 0.7.13 running on Android Pixel 3 (blueline), Android 10 (QQ1A.200105.002) API 29, ABI arm64-v8a, (Android/aosp_blueline/blueline:10/QQ1A.200105.002/2020.01.06.21:user/release-keys)
2020-01-27 10:06:25 Log cleared.
2020-01-27 10:16:47 Network Status: not connected
2020-01-27 10:16:47 Debug state info: not connected, pause: userPause, shouldbeconnected: false, network: PENDINGDISCONNECT 
2020-01-27 10:17:12 Network Status: BLOCKED  to WIFI 
2020-01-27 10:17:12 Debug state info: BLOCKED  to WIFI , pause: noNetwork, shouldbeconnected: false, network: DISCONNECTED
